scripts/workspace_cleanup/README.md

```diff
@@ -0,0 +1,39 @@
+# Workspace cleanup scripts
+
+The scripts in this directory can be used to delete Terra workspaces in bulk. Proceed with caution!
+
+`get_workspace_list.py` gathers a list of workspaces in a given project, along with information about storage cost and owners. That should be manually curated before passing it to the script to delete.
```

Comment:
"That" --> the output of the script should be manually created?

Reply:
Ah yes let me clarify. Yes, the output of that script is a csv and the user should manually curate/whittle it down to only the workspaces they want to delete (and only the `project` and `workspace` columns) to pass to the delete_workspaces script.